setup.py

```diff
@@ -31,11 +31,14 @@
     url="https://github.com/mlperf/logging",
     packages=setuptools.find_packages(),
     classifiers=[
-      "Programming Language :: Python :: 2",
-      "Programming Language :: Python :: 3",
-      "License :: OSI Approved :: Apache Software License",
-      "Operating System :: OS Independent",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: OS Independent",
     ],
     license='Apache 2.0',
     include_package_data=True,
+    install_requires=[
+        'pandas',
```

Comment:
Why do we need pandas ?

Reply:
This is because `result_summarizer` needs pandas. `result_summarizer` needs pandas because I want `pandas.Dataframe`'s capability to store tabular data once and output a lot of different types of format (including csv and human-readable), with us explicitly implementing a "format writer" as we used to for csv.